src/main.cc

```diff
@@ -51,7 +51,8 @@ void Serialization(std::ostream *stream,
     *stream << std::flush;
 }
 
-int ProcessAST(const Config& config, std::unique_ptr<std::istream>& in, std::unique_ptr<std::ostream>& out) {
```

Comment:
Yeah, are we happy with https://github.com/apiaryio/snowcrash/pull/380 ? If there isn't any problems I'd like to move forward with this format and I will recreate my PR from master.